### ISSUE TITLE
FPP MOON NOT CONNECTION ON IPHONE

### ISSUE THREAD

**Issue by GMANFRE57** [KulpLights/fpp-FPPMon#20]
My email is christmas4fun@gmail.com im having issue on my iphone is refresh all the time and not connecting (searching for fpp instances,,,) look ok on my pi.

**Comment by Bthomas1326** [comment]
same issue here.   my FPPmon plugin shows that FPPD is not running but FPPD is running on my controllers.

**Comment by Bthomas1326** [comment]
@GMANFRE57   I finally got mine working.  I had to unistall the plugin.   Reboot the controller.    Reinstall the plugin then restart FPPD.   Give that a try

**Comment by GMANFRE57** [comment]
Yes work now ThanksSent from my iPhoneOn Nov 7, 2024, at 11:53 AM, Bthomas1326 ***@***.***> wrote:﻿
@GMANFRE57   I finally got mine working.  I had to unistall the plugin.   Reboot the controller.    Reinstall the plugin then restart FPPD.   Give that a try
